Integrations/TruSTAR/TruSTAR_description.md

```diff
@@ -0,0 +1,3 @@
+# TruSTAR
+
+TruSTAR is an Intelligence Management Platform that helps you operationalize data across tools and teams, helping you prioritize investigations and accelerate incident response.
```

Comment:
this should be in the description section within the yml, this file is supposed to help users configure their instance within Demisto